public/styles/main.css

```diff
@@ -354,6 +354,10 @@ a.connections-table.icon {
   font-size: 1.5rem;
   max-height: 500px;
   min-height: 250px;
+  &:hover {
```

Comment:
I still think this should also apply to `focus-visible` so you can tab throug hthe items. Can you check if I'm right?